<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreItemSelectionDialog</class>
 <widget class="QDialog" name="StoreItemSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="CItemListView" name="tblItems"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CItemListView</class>
   <extends>QTableView</extends>
   <header>library/ItemListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
